js/mocha-tests/test/ES6Class.test.js

```diff
@@ -0,0 +1,24 @@
+// use named imports
+import { ES6Class } from '../src/ES6Class';
+
+// Use sinon to spy and stub
+describe('sinon', () => {
+  before(() => {
```

Comment:
I do not think that you can stub the method of an instantiated ES6 class with sinon. I know that you can instantiate the stub of a class:

```
sinon.createStubInstance(MyConstructor)
```

However this example shows how you can stub on the propotype of a class, which can be useful if you then inject the class into another component that will instantiate it.

If you want, I can add an example of `createStubInstance` (typical example is to stub a logger that you inject in a class).